pages/contexts/NotificationContext.tsx

```diff
@@ -0,0 +1,60 @@
+import { createContext, FunctionComponent, useState } from 'react';
+import { v4 as uuid } from 'uuid';
+
+import {
+  MailgunData,
+  NotificationContextInterface,
+  SendgridData,
+  SlackData,
+  SMTPData,
+  TeamsData,
+  TelegramData,
+  WebhookData,
+} from './NotificationContextInterface';
+
+type Data =
+  | SMTPData
+  | MailgunData
+  | SendgridData
+  | WebhookData
+  | SlackData
+  | TeamsData
+  | TelegramData;
```

Comment:
Just a thought, instead of rewriting everything here, can we import from `@hyperjumptech/monika`? Something like

```javascript
import { Notification } from '@hyperjumptech/monika/lib/interfaces/notification';
import { NotificationContextInterface } from './NotificationContextInterface';
```

Then monika-config-generator will always be in sync with monika.

Reply:
Ah, nice suggestion!